Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -13,10 +13,15 @@ private enum SearchListSection: Int {
     case Search
 }
 
+protocol SearchViewControllerDelegate: class {
```

Comment:
Traditionally, at least from what I've seen so far, delegates are associated with a view. If we made a `URLPickerDelegate`, who would own it? I would expect it to be associated with a `URLPicker`. Having protocols floating around in the ether, then arbitrarily associating them with classes because the classes match that behavior doesn't really seem like good design. We have the concept of a `HomePanel`, so it's paired with a `HomePanelDelegate`. We have a `SearchViewController`, so we pair it with a `SearchViewControllerDelegate`. Isn't that how we've defined all of our delegates so far?